src/utils/validateEmail.js

```diff
@@ -0,0 +1,13 @@
+const validateEmail = (email) => {
+  if (!email) {
+    return 'email is required';
+  }
+
+  const emailPattern = /^[\w.+-]+@([\w-]+\.){1,3}[\w-]{2,}$/;
+
+  if (!emailPattern.test(email)) {
+    return 'email is not valid';
+  }
+};
```

Comment:
Consider returning 'null' or 'undefined' explicitly when the email is valid. This will make it clear that the function returns a string only when the email is missing or invalid.